config/enabled.yml

```diff
@@ -524,6 +524,11 @@ Style/Documentation:
     - 'spec/**/*'
     - 'test/**/*'
 
+Style/DoublePipeEquals:
```

Comment:
I never heard this name before. 🙂 It should be `ConditionalAssignment`, but we have another cop with that name. 🤔 How about `OrAssignment` or `OrEquals`?